isis/tests/FunctionalTestsChan1m32isis.cpp

```diff
@@ -0,0 +1,558 @@
+  }
+  catch (IException &e) {
+    FAIL() << "Unable to ingest Chandrayaan image: " << e.toString().toStdString().c_str() << std::endl;
+  }
+  Cube cube(cubeFileName);
+  Pvl *isisLabel = cube.label();
+
+  // Dimensions Group
+  PvlGroup &dimensions = isisLabel->findGroup("Dimensions", Pvl::Traverse);
+  ASSERT_EQ(int(dimensions["Samples"]), 304);
+  ASSERT_EQ(int(dimensions["Lines"]), 5);
+  ASSERT_EQ(int(dimensions["Bands"]), 3);
+
+  // Pixels Group
+  PvlGroup &pixels = isisLabel->findGroup("Pixels", Pvl::Traverse);
+  ASSERT_EQ(pixels["Type"][0].toStdString(), "Real");
+  ASSERT_EQ(pixels["ByteOrder"][0].toStdString(), "Lsb");
+  ASSERT_EQ(double(pixels["Base"]), 0.0);
+  ASSERT_EQ(double(pixels["Multiplier"]), 1.0);
+
+  // Instrument Group
+  PvlGroup &inst = isisLabel->findGroup("Instrument", Pvl::Traverse);
+  ASSERT_EQ(inst["SpacecraftName"][0].toStdString(), "CHANDRAYAAN-1");
+  ASSERT_EQ(inst["InstrumentId"][0].toStdString(), "M3" );
+  ASSERT_EQ(inst["TargetName"][0].toStdString(), "MOON" );
+  ASSERT_EQ(inst["SpacecraftClockStartCount"][0].toStdString(), "2/1531046.542" );
+  ASSERT_EQ(inst["SpacecraftClockStopCount"][0].toStdString(), "2/1531047.050" );
+  ASSERT_DOUBLE_EQ(double(inst["LineExposureDuration"]), 101.76);
+  ASSERT_EQ(inst["StartTime"][0].toStdString(), "2008-11-29T17:14:29.729807" );
+  ASSERT_EQ(inst["StopTime"][0].toStdString(), "2008-11-29T17:14:30.238607" );
+  ASSERT_EQ(inst["SpacecraftYawDirection"][0].toStdString(), "FORWARD" );
+  ASSERT_EQ(inst["OrbitLimbDirection"][0].toStdString(), "DESCENDING" );
+
+  // Archive Group
+  PvlGroup &archive = isisLabel->findGroup("Archive", Pvl::Traverse);
+  ASSERT_EQ(archive["ProductId"][0].toStdString(), "M3G20081129T171431_V03_RDN" );
+  ASSERT_EQ(archive["SourceProductId"][0].toStdString(), "M3G20081129T171431_V01_L0.IMG" );
+  ASSERT_EQ(archive["ProductType"][0].toStdString(), "CALIBRATED_IMAGE" );
+
+  // BandBin Group
+  // Check size, first, 2 middle, and last values? Enough?
+  PvlGroup &bandbin = isisLabel->findGroup("BandBin", Pvl::Traverse);
+  ASSERT_EQ(bandbin["Center"].size(), 85);
+  ASSERT_EQ(bandbin["FilterNumber"].size(), 85);
+  ASSERT_EQ(bandbin["OriginalBand"].size(), 85);
+
+  ASSERT_DOUBLE_EQ(bandbin["Center"][0].toDouble(), 460.990);
+  ASSERT_DOUBLE_EQ(bandbin["Center"][21].toDouble(), 1009.95);
+  ASSERT_DOUBLE_EQ(bandbin["Center"][42].toDouble(), 1429.15);
+  ASSERT_DOUBLE_EQ(bandbin["Center"][84].toDouble(), 2976.20);
+
+  ASSERT_DOUBLE_EQ(bandbin["FilterNumber"][0].toDouble(), 5);
+  ASSERT_DOUBLE_EQ(bandbin["FilterNumber"][21].toDouble(), 57);
+  ASSERT_DOUBLE_EQ(bandbin["FilterNumber"][42].toDouble(), 99);
+  ASSERT_DOUBLE_EQ(bandbin["FilterNumber"][84].toDouble(), 253);
+
+  ASSERT_DOUBLE_EQ(bandbin["OriginalBand"][0].toDouble(), 1);
+  ASSERT_DOUBLE_EQ(bandbin["OriginalBand"][21].toDouble(), 22);
+  ASSERT_DOUBLE_EQ(bandbin["OriginalBand"][42].toDouble(), 43);
+  ASSERT_DOUBLE_EQ(bandbin["OriginalBand"][84].toDouble(), 85);
```

Comment:
Similar thought with the bandbins in the group. I don't think it's super important to test outside of the default test but I would like to hear other opinions/points regarding this.